### PR TITLE
Show helpful error for comments that are too long

### PR DESCRIPTION
**Pull request by baltpeter**
This is for https://github.com/datenanfragen/backend/pull/79. This PR should be deployed first.

### CODE REVIEW

File: src/Components/CommentsWidget.tsx [hunk 1]
```diff
@@ -226,8 +226,12 @@ export function CommentForm(props: CommentFormProps) {
                 ...(props.allowRating && rating ? { additional: { rating } } : {}),
             }),
         })
-            .then((res) => {
-                if (!res.ok) throw new Error('Unexpected response from comments server.');
+            .then(async (res) => {
+                const payload = await res.json();
+
+                if (payload.message?.includes('length must be less than')) {
```
[the remark this answers is not in the document]
Comment (baltpeter) [reply src/Components/CommentsWidget.tsx:232]: No.

Comment (mal-tee) [inline src/Components/CommentsWidget.tsx:232]: So we should do this instead?

```suggestion
                if (payload?.message?.includes('length must be less than')) {
```

Comment (baltpeter) [reply src/Components/CommentsWidget.tsx:232]: But `payload.message?` is already guarding against a payload potentially not having a message?

Your suggested change would guard against `payload` itself being `null` or `undefined`. If the backend is behaving correctly, we do always send a JSON payload with a `message` (https://github.com/datenanfragen/backend/blob/b_length-limits/src/handlers/comments/putComment.js#L11, https://github.com/datenanfragen/backend/blob/b_length-limits/src/handlers/comments/putComment.js#L32, https://github.com/datenanfragen/backend/blob/b_length-limits/src/handlers/comments/putComment.js#L36).  
The only way for `payload.message` to throw is if the response body was `null`. If that happens, the backend is clearly malfunctioning and we would drop into the `catch` anyway, which shows an appropriate error message.

**Review by mal-tee — APPROVED** [review]
Thank you!

**Pull request merged**